packages/components/src/components/chat/SuggestionGrid.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="suggestion-grid" v-if="suggestions.length">
-    <p>Not sure? Here are some useful starting points:</p>
+    <p>Here are some useful starting points:</p>
```

Comment:
I am on the verge of advocating for the removal of the prompts entirely, but while they still exist they should match the Navie metaphor a little more closely. Instead of 'starting points' it's more like 'ask Navie'. And we can drop the map pin icons as well. they distract from the primary interaction -- the text input.

Reply:
What's nice about the Prompts is that they give people a way to try out the AI without having to even think of a question. The Tips are designed for maybe later on, when they are asking their own questions and want to learn how to use Navie better. And, Navie is a thing that needs to be learned, like other AI tools - there are ways that it works well and ways that it doesn't work well. It's not a magic box.